replay/models/cql.py

```diff
@@ -131,9 +132,8 @@ class CQL(Recommender):
         impl (d3rlpy.algos.torch.cql_impl.CQLImpl): algorithm implementation.
```

Comment:
Do we have this argument?

Reply:
It can be accepted by the inner implementation, so yes.